packages/ai-chat/src/serverEntry.ts

```diff
@@ -74,6 +74,10 @@ export {
   BusEventCustomPanelOpen,
   BusEventCustomPanelPreClose,
   BusEventCustomPanelPreOpen,
+  BusEventWorkspaceClose,
```

Comment:
Interesting on the close events. My first thought was "if they are in charge of closing the workspace panel, what is this even for?" but I guess there are times when we might close it for them. Like if someone restarts the conversation.